keras_nlp/utils/preset_utils.py

```diff
@@ -333,6 +335,65 @@ def _validate_backbone(preset):
         )
 
 
+def create_model_card(preset):
+    model_card_path = os.path.join(preset, README_FILE)
+    markdown_content = ""
+
+    # YAML
+    markdown_content += "---\n"
+    markdown_content += "tags:\n"
+    markdown_content += "- keras-nlp\n"
+    markdown_content += "---\n"
+
+    config = load_config(preset, CONFIG_FILE)
+    model_name = (
+        config["class_name"].replace("Backbone", "")
+        if config["class_name"].endswith("Backbone")
+        else config["class_name"]
+    )
+    model_link = f"https://keras.io/api/keras_nlp/models/{model_name.lower()}"
```

Comment:
Is `model_name.lower()` always correct? Can you try it on all Model instances found in the namespace?

Reply:
Right! It doesn't work for some models!
I was trying to figure out if I can programatically convert the model name to model path, e.g. converting CamelCase to snake_case, but things like `XLMRoberta` -> `xlm_roberta` makes it complicated! I can create a manual mapping in a dictionary from model to path based on [this](https://github.com/keras-team/keras-io/blob/master/scripts/nlp_api_master.py) which seems to be our source of truth! But then kerasNLP and keras.io won't be automatically synced if we change one!

Reply:
Fixed this issue. I believe it should now work for all models.